nemo/lightning/pytorch/callbacks/flops_callback.py

```diff
@@ -32,6 +34,7 @@
     "nemotron": flops_formulas.nemotron,
     "mixtral": flops_formulas.mixtral,
     "bert": flops_formulas.bert,
+    "hyena": hyena,
```

Comment:
Can we just add this to flop formulas and keep it consistent with the above?

Reply:
My goal here was to keep the arc+nvidia copyright header out of our shared flops formula file.